hc/settings.py

```diff
@@ -49,6 +49,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'hc.accounts.middleware.TeamAccessMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
@briankabiro This is a revert PR, so these changes do not belong in this branch...as stated on the description. Undo this so we can merge the revert